### PR TITLE
Notebooks Update

### PR DESCRIPTION
**Pull request by Yves-Noel**
Python file added in the test section.

### CODE REVIEW

File: test/app/Intro Notebook.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+
+get_ipython().system('pip install quandl')
+
+
+# In[3]:
+
+
+import quandl
+quandl.ApiConfig.api_key = '38VWdHpGVLZs94hZCMC1'
```
Comment (joewood) [inline test/app/Intro Notebook.py]: Can you remove the API key from the code. Instead you should use an environment variable using:

```python
import os
quandl.ApiConfig.api_key =  os.environ["QUANDL_API_KEY"]
```

**Review by joewood — CHANGES_REQUESTED** [review]
Looks good other than the API. Remember to set the environment variable in your shell once it's setup. You shouldn't even check-in API Keys. 

Also can you remove the whitespace lines in this file?

**Review by Yves-Noel — COMMENTED** [review]
Made a minor update on line 7 and 8
Cleaned code by deleting whitespace

**Review by joewood — COMMENTED** [review]
Great thanks, merging this file.

**Pull request merged**